spec/music_album_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
+require_relative '../classes/item/music_album.rb'
+
+describe MusicAlbum do
+  let(:album) { MusicAlbum.new('2010-01-01', true) }
```

Comment:
Please initialize the MausicAlbum with a date as MusicAlbum.new(Date.new(2010, 1, 1), true) so publish_date can be appropriately recognized as a date object.